Работаем с условиями

andrewk990 opened the pull request:
Проверка условия module1-task2.

---
:mortar_board: [Работаем с условиями](https://up.htmlacademy.ru/php/12/user/1418251/tasks/4)

andrewk990 committed:
Проверка условия module1-task2.

nkolosov commented on index.php:
```diff
@@ -30,6 +30,23 @@
 
         <!-- здесь должен быть PHP код для показа меню и данных пользователя -->
 
+        <?php if ($is_auth == 1) : ?>
```
1. Хромает форматирование (отступы на разных строках разные)
2. Сравнение нужно делать строго-типизированным (использовать оператор `===`)

nkolosov commented on index.php:
```diff
@@ -30,6 +30,23 @@
 
+        <?php if ($is_auth == 1) : ?>
+         <div class="user-menu__logged">
+    		<p>#user_name#</p>
```
В задании сказано, что вместо этой строки нужно вывести значение переменной

andrewk990 committed:
Исправление замечаений к ПР:
- Хромают отступы, сместил div в условии.
- Строго-типизированное сравнение, выяснил для чего, поставил, намотал на ус =)
- Заменить вывод на переменную: выполнил, спасибо, буду стараться внимательнее читать задание.
- Readme.md добавил информацию о кураторе, ИФ, ссылку на профиль.

andrewk990 commented:
Исправление замечаений к ПР:
- Хромают отступы, сместил div в условии.
- Строго-типизированное сравнение, выяснил для чего, поставил, намотал на ус =)
- Заменить вывод на переменную: выполнил, спасибо, буду стараться внимательнее читать задание.
- Readme.md добавил информацию о кураторе, ИФ, ссылку на профиль.
